commands/apply.js

```diff
@@ -1,32 +1,68 @@
 const clans = require('../data/clan-data');
+const positions = require('../data/staffteam-data');
 
 module.exports.execute = async (client, message, args) => {
 	const searchTerm = args.join(' ').toLowerCase();
 
 	if (searchTerm.length === 0) {
 		return await message.channel.send(
-			'❌ Please specify the clan you wish to apply for.'
+			'❌ Please specify the clan or staff position you wish to apply for.'
 		);
 	}
 
 	let clan = clans.find((c) => c.names.includes(searchTerm));
-
-	if (!clan) {
-		return await message.channel.send(
-			`❌ The clan \`${searchTerm}\` couldn't be found.`
+	let position = positions.find((p) => p.names.includes(searchTerm));
+	let isClan = false;
+	let isPosition = false;
+	if (clan) {
+		isClan = true;
+	}
+		if (position) {
+		isPosition = true;
+		}
+	if (!isPosition && !isClan) {
```

Comment:
These lines can be cleaned up:
```
if (!position && !clan)
```
similar to the conditions used later in the function